with-jotai/atoms.ts

```diff
@@ -0,0 +1,5 @@
+import { atom } from "jotai";
+
+const counterAtom = atom<number>(0);
+
+export default counterAtom;
```

Comment:
I think we should either export counterAtom without `default` (since the name of this file indicate there are many atom inside), OR merge this into the `counter.tsx` view.

For simplicity's shake tho, I recommend the 2nd option

Reply:
We can merge this into 'counter.tsx' file itself.